src/media-segment-request.js

```diff
@@ -177,7 +177,7 @@ const handleKeyResponse = (segment, finishProcessingFn) => (error, request) => {
  *                                        this request
  */
 const handleInitSegmentResponse =
-(segment, captionParser, finishProcessingFn) => (error, request) => {
+({segment, captionParser, handlePartialData, finishProcessingFn}) => (error, request) => {
```

Comment:
Was this used anywhere?

Reply:
This is to allow for partial appends of fmp4s

Reply:
It doesn't look like it's used in this function though.

Reply:
Yeah, not quite yet. It will eventually be passed into the CaptionParser though. I thought the change in contract (three arguments to one object argument) would be better to do now rather than later though.